Dockerfile

```diff
@@ -1,12 +1,12 @@
 ############# builder
-FROM golang:1.14.9 AS builder
+FROM  mirror.gcr.io/library/golang:1.14.9 AS builder
```

Comment:
It will break this release script -ref https://concourse.ci.gardener.cloud/teams/gardener/pipelines/gardener-extension-os-gardenlinux-master/jobs/master-release-job/builds/3

Reply:
so let's wait for the decision whether we also change g/g?

Reply:
yes, I think we can wait for the finial decision and do the unified change.

Reply:
Also, this won't work, see https://cloud.google.com/container-registry/docs/pulling-cached-images:

> Pulling cached images does not count against Docker Hub rate limits. However, there is no guarantee that a particular image will remain cached for an extended period of time. Only obtain cached images on mirror.gcr.io by configuring the Docker daemon. A request to pull directly from mirror.gcr.io will fail if a cached copy of the image does not exist.

For example:
```
$ docker pull mirror.gcr.io/library/golang:1.14.9
Error response from daemon: manifest for mirror.gcr.io/library/golang:1.14.9 not found: manifest unknown: Failed to fetch "1.14.9" from request "/v2/library/golang/manifests/1.14.9".
```

Reply:
thx. I reverted the mirror.